### DISCUSSION TITLE
Export whole wiki.js and import again?

### DISCUSSION THREAD

**Discussion by renetsk** [requarks/wiki#6449] [Help / Questions]
Hello, using portainer / docker / wiki.js. Just found I have not ideal docker compose (see attachment).
Setuped official docker compose with postgresql.
Is there any way to export everything from "old" wiki to "new" wiki?

old compose
![old-wiki](https://github.com/requarks/wiki/assets/121858379/987b0607-e935-43cf-91c7-77f32a306260)

new compose
![new-wiki](https://github.com/requarks/wiki/assets/121858379/872fc39f-92cd-4cac-9138-e028a162ba1f)

**Comment by P4mes4n** [comment]
In the Admin Panel -> Storage -> Local File System you should be able to setup a local Backup Destination, then using "Create Backup" should create a file in the destination containing all configuration and content of your wiki.
In your new Wiki create the same configuration, place your backup file in the folder of your new wiki and choose "Import Everything" in your Local File Storage Setup.

**Reply by cesarnr21** [reply]
Hi, when I run "Create Backup" only an empty `tar.gz` file is created in the `_manual` folder. I also checked the `_daily` folder for daily backups, and only one page was saved that I left open overnight. It wouldn't make sense for pages to need to be open and loaded for them be included in the backup.

**Reply by NGPixel** [reply]
@cesarnr21 You need to run the "Dump all content to disk" action when setting up the module the first time. Otherwise only changes from this point forward will be included.

**Reply by cesarnr21** [reply]
@NGPixel okay. That seemed to work, except that the contents where saved as markdown files and weren't compressed (this is fine).

Does the daily backup functionality create a new file everyday with all of the wiki content and then also the changes made on that day? And from now on, should any manual backup include all of the content from the wiki, or I always need to use the "Dump all content to disk"?

btw: really enjoy using wiki js so far. Looking forward to contributing in the future.

**Reply by NGPixel** [reply]
@cesarnr21 That's not how the daily backup option works. The zip archive is made from the files found in the folder, not from what's in the DB. After running the "Dump all content to disk" action, you can run the "Create Backup" action to create an archive from it. Or you can just wait for the next daily backup which will take what's currently on disk.

When you make changes, the changes are also saved to disk. The daily backup will then create an archive from these files. Because you already had pages and enabled the module afterwards, these pages were not initially included.

**Reply by cesarnr21** [reply]
@NGPixel Okay, now I see how it works, thank you.
